modules/http-api/src/main/java/org/mule/runtime/http/api/ws/WebSocketCloseCode.java

```diff
@@ -56,20 +100,52 @@ public enum WebSocketCloseCode {
   public static WebSocketCloseCode fromProtocolCode(int protocolCode) {
```

Comment:
Under heavy usage, these logs could generate excessive IO usage. Please change the log levels to DEBUG